pages/admin/lessons/dojo/modules.tsx

```diff
@@ -0,0 +1,126 @@
+import { gql, useQuery } from '@apollo/client'
+import { GetAppProps, withGetApp } from '../../../../graphql'
+import { get, sortBy } from 'lodash'
+import React, { useEffect, useMemo, useState } from 'react'
+import AdminLessonNav from '../../../../components/admin/lessons/AdminLessonNav'
+import AdminLessonSideNav from '../../../../components/admin/lessons/AdminLessonSideNav'
+import AdminModuleInputs, {
+  Props
+} from '../../../../components/admin/lessons/AdminModuleInputs'
+import Breadcrumbs from '../../../../components/Breadcrumbs'
+import styles from '../../../../scss/modules.module.scss'
+import { AdminLayout } from '../../../../components/admin/AdminLayout'
+
+const MODULES = gql`
+  query {
+    modules {
+      id
+      name
+      content
+      lesson {
+        id
+      }
+    }
+  }
+`
+
+type Module = {
+  id: number
+  name: string
+  content: string
+  lesson: { id?: number }
+}
+type Modules = Module[]
+
+type ModulesTabProps = {
+  filteredModules: Modules
+  lessonId: number
+  refetch: Props['refetch']
+}
+
+const ModulesTab = ({
+  filteredModules,
+  lessonId,
+  refetch
+}: ModulesTabProps) => {
+  const [selectedIndex, setSelectedIndex] = useState(-1)
+  const onAddItem = () => setSelectedIndex(-1)
+  const onSelect = (item: Module) => setSelectedIndex(item.id)
+
+  useEffect(() => setSelectedIndex(filteredModules[0]?.id), [lessonId])
```

Comment:
For the sake of consistency, can we stick to using either `get` or `optional chaining`?
Also, can you try setting a default value as empty object for filteredModules in this function's arguments? That should avoid the issue of having to use optional chaining or lodash `get` here. I'm not sure if it can be done without typescript issues tho